src/main/java/org/jenkinsci/plugins/saml/SamlEncryptionData.java

```diff
@@ -0,0 +1,34 @@
+package org.jenkinsci.plugins.saml;
```

Comment:
All java sources in this repo contain a header with license information. I guess this practice should be applied to this file as well.